internal/serve/serve.go

```diff
@@ -57,6 +59,9 @@ type Configs struct {
 	HorizonClientURL                   string
 	DistributionAccountSignatureClient signing.SignatureClient
 	ChannelAccountSignatureClient      signing.SignatureClient
+
+	// Error Tracker
+	AppTracker sentry.SentryTracker
```

Comment:
Here you should use the `apptracker.AppTracker`

Reply:
Done